.github/workflows/deploy.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         run: |
             SKIP_BUILD=1 just docker-serve prod -d
             sleep 5
-            just docker-smoke-test || { docker logs job-server_prod_1; exit 1; }
+            just docker-smoke-test || { docker logs job-server-prod-1; exit 1; }
```

Comment:
Could this be from when we moved to docker compose v2? That was June 2024, but I can't see logs for actions that far back :shrug:

Reply:
that's my guess, yes!